modules/ximgproc/src/structured_edge_detection.cpp

```diff
@@ -419,7 +419,16 @@ class StructuredEdgeDetectionImpl : public StructuredEdgeDetection
     /*!
      * This constructor loads __rf model from filename
      *
-     * \param filename : name of the file where the model is stored
+     * Refer to <https://github.com/opencv/opencv_extra/blob/master/testdata/cv/ximgproc/model.yml.gz>
```

Comment:
I have changed all the code in  opencv_contrib to use the new `create` API, because
the `CV_DEPRECATED` results in compilation error in the buildbot for python wrappers.
The old API is now deleted.